pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -190,6 +281,8 @@ def __call__(self, time: float):
                             [InfectionStatus.InfectASympt,
                              InfectionStatus.InfectMild,
                              InfectionStatus.InfectGP]:
-                        self.set_infectiousness(person)
+                        self.set_infectiousness(person, time)
                     self._update_next_infection_status(person)
                     self._update_time_status_change(person, time)
+                    continue
```

Comment:
I think this is being misused - continue skips to the next iteration of the top level for/while loop. Here it is at the end of a while loop, so won't do anything. Do you also want to skip on to the next person?

Reply:
I removed it